src/lib/ng-package/entry-point/write-package.transform.ts

```diff
@@ -138,19 +138,23 @@ async function writePackageJson(
 
   // Removes scripts from package.json after build
   if (packageJson.scripts) {
-    if (pkg.keepLifecycleScripts !== true) {
-      log.info(`Removing scripts section in package.json as it's considered a potential security vulnerability.`);
-      delete packageJson.scripts;
+    if (pkg.keepPackageScripts || pkg.keepLifecycleScripts) {
+      const npmPublishInfo = isIvy
```

Comment:
I don't follow the need to a different error message when using Ivy.

Reply:
I'll revert to using just one message.

--- 

If I remember correctly, this is a legacy from #1711 where after my changes you were allowed to publish Ivy libraries, but after the discussions we dropped some of those changes and I created this MR.